### DISCUSSION TITLE
Machine Specs

### DISCUSSION THREAD

**Discussion by mcl0vinit** [planetscale/discussion#505] [Q&A] [answered]
What are the machine specs for each tier? I see that scaler pro has it described using the slider but I was wondering how hobby and scaler plans match up to make sure I select the correct size machine for my needs if I need to upgrade to scaler pro.

**Comment by jgreet** [comment, marked as answer]
Hello!

The various cluster sizes are outlined in our docs here:
https://planetscale.com/docs/concepts/billing#planetscale-plans

Production Scaler plan branches use the PS-10 cluster size. They have one replica, while production Scaler Pro branches have two.

Production Hobby branches have about half the CPU and memory resources of the PS-10 cluster size. These branches also have a single replica.
